src/main/java/com/nvidia/spark/rapids/jni/JSONUtils.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.jni;
+
+import ai.rapids.cudf.*;
+
+public class JSONUtils {
+  static {
+    NativeDepsLoader.loadNativeDeps();
+  }
+
+  public enum PathInstructionType {
+    SUBSCRIPT,
```

Comment:
nit: I would prefer to have the enum to int mapping be specific instead of using the ordinal. It is minor, but helps to prevent issues where one is updated and the other is missed.

Reply:
will do in a follow-on